integration_tests/steps/jira_issues_tests.py

```diff
@@ -255,13 +255,16 @@ def device_statistics(context):
     counter = 0
     for house in ["House 1", "House 2"]:
         for device in output_dict[house]:
+            print(f"device: {device}")
```

Comment:
Did not see this before, please make sure that you remove the prints before merging